lutris/util/display.py

```diff
@@ -130,37 +130,121 @@ def get_display_manager():
 USE_DRI_PRIME = len(_get_graphics_adapters()) > 1
 
 
-def get_compositor_commands():
-    """Nominated for the worst function in lutris"""
+# Pretend this is an enum.
+class DE:
```

Comment:
Why pretend? Just make it a proper enum: https://docs.python.org/3.6/library/enum.html#creating-an-enum

Reply:
I did not use enum.Enum because afaict nothing else in lutris uses the enum module, but if you say it's okay I'll make a commit that converts this class to an Enum.